### PR TITLE
feat(web): chat queue panel with autonomous server-side drain

### LINKED ISSUE #46: Steering/queue support in chat composer
### Problem / motivation

pi supports a steering/queue mechanism where users can send follow-up instructions while the assistant is still running (e.g., "also check the tests for edge cases"). pi-web currently does not expose this — once a message is sent, the composer is locked until the response completes. Users have no way to steer or queue additional instructions.

### Proposed solution

Add **steering/queue** support to the chat composer:

1. While the assistant is running (status = "running"), the chat composer remains enabled and shows a secondary "Steer" or "Queue" button alongside (or replacing) the send button.
2. Typing a message and pressing Enter (or clicking Steer) sends an RPC steer/queue command to the active worker.
3. The sent instruction appears in the conversation stream as a steering entry (distinct visual style from regular user messages).
4. If the worker is not yet accepting steering, the message is queued and sent as soon as the worker transitions to the accepting state.
5. Status indicator updates to show "steering..." or "queued (n pending)".

**Backend:**
- New RPC method or API endpoint: `POST /api/sessions/:id/steer` with the steering message.
- Worker lifecycle handling: detect when the pi worker is in a steerable state, queue messages if not.
- New entry type in JSONL for steer/queue entries.

**Scope:**
- Frontend: `web/src/session/chat/chat-composer-runner.js` — enable composer during running state, add steer button and logic.
- Frontend: `web/src/session/chat/chat-api.js` — add `steerChat()` method.
- Frontend: `web/src/session/render/session-entry-renderer.js` — render steering entries with distinct styling.
- Backend: `internal/server/` — new endpoint + RPC handling.
- Tests: unit + E2E.

**PR requirements:**
- E2E tests covering: sending a steer message during a running chat, verifying it appears in the session, and the assistant responds to the steer.
- Screenshots showing the steer flow.

### Alternatives considered

_No response_

### Acknowledgement

- [ ] I understand the maintainer triages requests and will invite a PR only if this moves forward.

### PR DESCRIPTION
**Pull request by setkyar**
Closes #46.

## Summary

Adds full **steering + queue** support to the chat composer, as requested in #46:

- **Steer** a follow-up message into the active turn (uses pi's existing `streamingBehavior:"steer"`).
- **Queue** N messages to run sequentially, drained autonomously by the backend.
- A docked panel above the composer with **pause/resume**, keyboard navigation, and per-row delete/send-now/edit actions.

The queue is **server-owned** (SQLite-backed via `internal/chatqueue`) and an autonomous drainer goroutine sends the next item as soon as the worker is idle — so queued work runs to completion even if you close the browser.

## Screenshots

**Steer in flight** — the chip shows above the composer while pi folds the message into the active turn:

steer chip mid-flight

**Queue stacked** — multiple queued messages, "auto-sending" hint that the server will drain them:

queue with 3 items

**Keyboard focus** — `↑↓` navigates, `E` edits, `⌫` deletes, `↩` sends-now, `Esc` exits:

keyboard focused row

**Paused** — `Pause` stops auto-draining; queue items survive a refresh / browser close until resumed:

paused queue

## What's in the PR

**Backend (Go)**
- `internal/chatqueue` package: SQLite `chat_queue_items` + `chat_queue_state` tables, monotonic per-session positions that never reuse, CRUD + `PopHead` for the drainer.
- `internal/server/chat_queue.go`: `GET/POST/DELETE/PATCH /api/chat/queue?id=...` with SSE `queue` broadcast on every mutation.
- `internal/server/chat_queue_drainer.go`: a single goroutine that kicks on queue mutations, on worker `running→idle` transitions, and every 5s; dispatches the head queued item through the existing `chatSender.Send`, spawning a worker if one isn't alive.
- WAL + `busy_timeout` on the shared SQLite handle so the drainer's reads don't block POSTs.

**Frontend (Svelte 5)**
- New `QueuePanel.svelte` floats above the composer card; mirrors `.pi-chat-shell`'s width/centering so it never sprawls when the sidebar is hidden.
- `QueueStore` (Svelte 5 `$state` class) hydrates from `GET /api/chat/queue` on mount and refetches on the SSE `queue` event (cross-tab + drainer sync).
- Document-level keyboard shortcuts — **↑↓** navigate, **⌫** delete, **↩** send-now, **E** edit-to-textarea, **Esc** exit — wired through capture phase so they beat the textarea's own Enter handler; fall through to the textarea the moment any content is typed.
- Steers stay browser-only (they belong to the in-flight stream); their chips clear the instant pi echoes the user entry back via live reload (precise text match + FIFO fallback for decorated content).
- 13 non-English locales drafted via `pi --provider opencode-go --model deepseek-v4-pro` (`scripts/translate_new_locale_keys.py`).

**Misc fixes alongside**
- Session-page first-paint: split scratchpad off the critical path and added a hover-prefetch for `/api/session` so opening a session feels instant.
- Live-events: `pi-session-reload` now dispatches **after** entries reconcile (was firing before the fetch resolved, so listeners saw stale state — same bug that made the steer chip linger).

## Test plan

- [x] `make check` — frontend tests, Go tests, vet, lint, format-check.
- [x] `make e2e` — 478 tests pass across 7 browser projects (Desktop Chrome/Firefox/Safari, Mobile Chrome/Safari, iPad portrait + landscape).
- [x] Unit: `chatqueue_test.go` (CRUD + position monotonicity + drainer rules), `chat_queue_test.go` (HTTP roundtrips), `chat_queue_drainer_test.go` (dispatch, pause-skip, running-skip, non-blocking kick).
- [x] Unit: `queue-store.test.js`, `steer-queue.test.js` — 25 cases covering hydration, send-now/edit DELETE path, pause/resume, steer chip lifecycle + cleanup.
- [x] E2E: `e2e/tests/steer-queue.spec.ts` — 14 cases covering steering, dismissal, queuing, pause/resume, keyboard shortcuts, panel placement/alignment, server persistence (**including a "browser close + new context, all client storage cleared" test that proves the queue survives**), and the new "steer chip clears the moment pi echoes the user entry" check.
- [x] Screenshots of all four queue/steer states (above), re-generable via `e2e/tests/_screenshots.spec.ts`.

### CODE REVIEW

File: scripts/translate_new_locale_keys.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    return out
+
+
+def js_escape(value: str) -> str:
+    """Match the project's single-quoted style: only ' and \\ need escaping."""
+    return value.replace("\\", "\\\\").replace("'", "\\'")
+
+
+def build_prompt(code: str, native: str, references: dict[str, str]) -> str:
+    ref_lines = "\n".join(
```
[the remark this answers is not in the document]
Comment (setkyar) [reply scripts/translate_new_locale_keys.py]: Fixed in dea96f6 — dropped the unused `ref_lines` block; the `pairs` list directly below was the one actually fed into the prompt.
